packages/map/src/popup/index.module.scss

```diff
@@ -0,0 +1,6 @@
+@import "~nebenan-ui-kit";
+
+.root:global(.mapboxgl-popup-content) {
```

Comment:
does it work? mapbox usually moves popup up to body node

Reply:
Yes it works because I pass the className into the MapboxPopup constructor

Reply:
No, it's wrong. Sorry